text/0016-katello-as-a-plugin/0020-remove-capsule.md

```diff
@@ -0,0 +1,48 @@
+- Feature Name: Remove Capsule in favor of Smart Proxy
+- Start Date: 2016-11-10
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+A Capsule is essentially a Smart Proxy that deploys Pulp, and a reverse proxy to achieve content federation and an isolated communication point fo clients. Foreman deploys the Smart Proxy on port 8443 by default, while Katello deploys it on 9090 by default. Katello deployed Smart Proxies on 9090 because Candlepin was already running on 8443 on the server.
+
+# Motivation
+[motivation]: #motivation
+
+The "Capsule" terminology and code creates a perceived difference between a Foreman Proxy and the Capsule even though a Capsule is a Foreman Proxy with a few added features. Centralizing development of the Foreman Proxy will allow better innovation and less confusion for community users. The expected outcome is that we've removed the notion of a 'Capsule' and are centralized on the Foreman Proxy and adding features to it.
```

Comment:
Yes, this is confusing as well. Everything in the Foreman core is "Smart Proxy", but in packaging the file is named "Foreman Proxy". There is no easy way out, I'd suggest to create new official name "Foreman Smart Proxy" and use this in documentation and Web UI, so we can keep both, but it will "teach" users that this is essentially the same thing.